parmed/amber/_amberparm.py

```diff
@@ -328,6 +332,22 @@ def from_structure(cls, struct, copy=False):
             # Maybe it just has CHARMM parameters?
             raise TypeError('AmberParm does not support all of the parameters '
                             'defined in the input Structure. Try ChamberParm')
+
+        # Convert all RB torsions to propers and delete the originals.
+        for dihedral in struct.rb_torsions:
+            proper_types = DihedralTypeList.from_rbtorsion(dihedral.type)
+            for proper_type in proper_types:
+                proper = _copy.copy(dihedral)
+                proper.type = proper_type
+                struct.dihedrals.append(proper)
+                struct.dihedral_types.append(proper.type)
+                proper.type.list = struct.dihedral_types
+
+        del struct.rb_torsions[:]
+        struct.rb_torsion_types.clear()
```

Comment:
`TrackedList` inherits from `list`, and `list.clear` was added in Python 3.  Remove this line to fix the Python 2 errors. (the `del struct.rb_torsions[:]` should handle the clearing)